src/shogun/regression/LinearRidgeRegression.cpp

```diff
@@ -75,24 +73,19 @@ bool CLinearRidgeRegression::train_machine(CFeatures* data)
 	for (int32_t i=0; i<num_feat; i++)
```

Comment:
can you add the ridge after you have computed the covariance below ? Makes more sense to me